config.toml

```diff
@@ -11,5 +11,105 @@ highlight_code = false
 # Whether to build a search index to be used later on by a JavaScript library
 build_search_index = false
 
+default_language = 'en'
+
+# languages = [
+#   {code = "fr"},
+# ]
+
+[translations]
+  [translations.en]
+
+  # base.html
+  lemmy_link_agg = "A link aggregator for the fediverse"
+  docs = "docs"
+  code_of_conduct = "Code of Conduct"
+  coc = "CoC"
+  contact = "Contact"
+  made_using_1= "Made using"
+
+  # index.html
+  link_aggregator_fediverse = "A link aggregator for the fediverse."
+  join_a_server = "Join a Server"
+  run_a_server = "Run a Server"
+  follow_communities = "Follow communities anywhere in the world"
+  is_similar_to_sites = "is similar to sites like"
+  you_subscribe_to_create = "you subscribe to communities you're interested in, post links and discussions, then vote and comment on them. Lemmy isn't just a reddit alternative; its a network of interconnected communities ran by different people and organizations, all combining to create"
+  a_single_frontpage = "a single, personalized front page"
+  of_favorite_memes = "of your favorite news, articles, and memes."
```

Comment:
Hmm the way these strings are split is not gonna work at all with languages that use a different word order than English (so most non-European languages). You cant put any formatting for bold or links into the string?

Reply:
Yup that's the main limitation. i18n can do this ( and we use this a lot in lemmy), but zola doesn't have a proper translation component for templates that can handle it. Its either this, or no formatting / links within sentences.

Reply:
Could you put Markdown in the translation text for formatting, and convert that to HTML at build time?